### ISSUE TITLE
Bug report: Unreadable tags due to insufficient colour contrast

### ISSUE THREAD

**Issue by 0xKai27** [ethereum/ethereum-org-website#9128]
### Describe the bug

The tags which are used on the dapps page to differentiate dapp categories does not have sufficient contrast resulting in them become unreadable.

Propose to keep render the tag text in black/white and vary the tag background colour according to category

### To reproduce

1. Go to the "Editors' choices' section on https://ethereum.org/en/dapps/
2. View the suggested cards
3. Try to read the tags

### Expected behavior

The tag text should be easily readable and differentiated based on different background color

### Screenshots

![image](https://user-images.githubusercontent.com/104422522/211448832-524dcbba-a163-421e-9bbb-de3500278b16.png)

### Desktop (please complete the following information)

- Browser: Chromium (Tested on Brave and Chrome)

**Comment by 0xKai27** [comment]
Similar issue on the Stablecoins page

<img width="1451" alt="image" src="https://user-images.githubusercontent.com/104422522/211525516-7530f705-3976-4c88-ac6e-a00ea132ed9b.png">

**Comment by amateurCrypt** [comment]
Hey can you assign this task to me?

**Comment by corwintines** [comment]
Thanks for reporting @0xKai27 

Thanks for volunteering @amateurCrypt

This seems to have been fixed with the migration of the Pill component in https://github.com/ethereum/ethereum-org-website/pull/8601 so im going to close the issue.
